test/libsolidity/syntaxTests/controlFlow/modifiers/modifier_override.sol

```diff
@@ -0,0 +1,17 @@
+contract A {
+	function f() mod internal returns (uint[] storage) {
+	}
+	modifier mod() virtual {
+		revert();
+		_;
+	}
+}
+contract B is A {
+	modifier mod() override { _; }
+	function g() public {
+		f()[0] = 42;
+	}
```

Comment:
This part is actually irrelevant for the test and could be removed... it's a demonstration of this being bad without an error, but for the test case, it's a bit distracting. But also fine with keeping it.